pkgs/by-name/su/surge-XT/package.nix

```diff
@@ -27,6 +31,21 @@ stdenv.mkDerivation rec {
     hash = "sha256-4b0H3ZioiXFc4KCeQReobwQZJBl6Ep2/8JlRIwvq/hQ=";
   };
 
+  patches = [
+    ./clap-option.diff
```

Comment:
Then please use `fetchpatch` with the [merge commit](https://github.com/surge-synthesizer/surge/commit/7bc191a67a3fb1f96adca7d9dacdb36ddd2b829f), and add a comment explaining the patch, and linking to the PR.